src/complexity_measures/missing_dispersion.jl

```diff
@@ -0,0 +1,65 @@
+using DelayEmbeddings
+export MissingDispersionPatterns
```

Comment:
I would argue that the source code for this is completely generic and doesn't depend on the estimator. Hence, we could define the generic **function** `missing_symbols` that does what `count_nonoccurring(x::AbstractVector{T}, est)` does. (I prefer this name over the name `count_nonoccuring` but we can debate further the name). Then, we can make this function as part of the public API, that is discussed in the same `@docs` block as `alphabet_length`. 

Then, you could keep `MissingDispersionPatterns` but in its docstring say clearly that "this function is just `missing_symbols(x, Dispersion())`, but it also has an implementation for the `complexity` interface due to its use as a complexity measure."

Reply:
This is a good idea, and I like the name `missing_symbols`.

There are other methods in the literature too that uses the presence/absence of symbols for characterisation of complexity. "Missing *permutation* patterns is one such method.

`missing_symbols` could then, without any extra effort from code maintainers or users, also potentially be used to explore complexity characterisation for any other discretisation scheme (i.e. `ProbabilitiesEstimator`s) for which that would be meaningful. 

I'll update the PR accordingly.